wsdl/ver10/schema/metadatastream.xsd

```diff
@@ -291,6 +291,7 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
 			<xs:element name="Extension" type="tt:FrameExtension" minOccurs="0"/>
 			<xs:element name="SceneImageRef" type="xs:anyURI" minOccurs="0"/>
 			<xs:element name="SceneImage" type="xs:base64Binary" minOccurs="0"/>
+			<xs:element name="AspectRatioTranformation" type="tt:AspectRatioTransformation" minOccurs="0"/>
```

Comment:
Suggest to add the same as optional element inside object also, to keep it consistent with "ObjectTrack" Event.